src/extension.ts

```diff
@@ -255,6 +254,15 @@ export function activate(context: VSCode.ExtensionContext) {
     }
   );
 
+  VSCode.commands.registerCommand('SonarLint.DeactivateRule', (ruleKey: string) => {
+    const configuration = getSonarLintConfiguration();
+    const rules = configuration.has('rules') ? configuration.get('rules') : {};
+    rules[ruleKey] = {
+      level: 'off'
+    };
+    return configuration.update('rules', rules, VSCode.ConfigurationTarget.Global);
```

Comment:
What is the purpose of `VSCode.ConfigurationTarget.Global` ?
I see in the file that we get the SonarLint config from the workspace `VSCode.workspace.getConfiguration('sonarlint')`, there is no equivalent to update the config (in the workspace instead of Global) ? Or maybe workspace and global are the same thing ?

Reply:
This asymmetry comes from the API itself: the configuration is read from the `workspace` object, which merges the user, workspace and folder configurations (when appropriate). On write, it's up to the extension to choose where to store the settings - and in the scope of this sprint, we want to store it in the global user settings.